app/controllers/donor_controller.rb

```diff
@@ -9,10 +9,19 @@ class DonorController < ApplicationController
   def profile
     @user = current_user
     @profile = DonorProfile.find_by(donor_id: current_user.id)
-    @donations = Donation.where(donor_id: current_user.id)
+    @donations = Donation.where(donor_id: current_user.id).includes(food_transaction: [{ recipient: :recipient_profile }, :coordinator])
+
     @pending_donations = @donations.where(status: Donation.type_pending)
     @inprogress_donations = @donations.where(status: Donation.type_in_progress)
     @completed_donations = @donations.where(status: Donation.type_completed)
+
+    gon.donations = @donations.as_json({
+      include: [{ food_transaction: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.